sys/include/net/ble.h

```diff
@@ -0,0 +1,405 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_ble General BLE defines
```

Comment:
I think "Bluetooth Low Energy (BLE)" is easier to find in the list of modules. At least I wouldn't search under "G" for the *B*luetooth definitions ;-)